src/database/PlayerInstance.ts

```diff
@@ -5,6 +5,8 @@ import { Player } from "@prisma/client";
 // wrapper class for Player
 // todo bad naming
 export class PlayerInstance {
+  private static instances: PlayerInstance[] = [];
```

Comment:
this isn't needed, GameInstance already keeps track of all of the PlayerInstances.

Reply:
Done